editor.planx.uk/src/theme.ts

```diff
@@ -226,11 +227,23 @@ const getThemeOptions = (primaryColor: string): ThemeOptions => {
   return themeOptions;
 };
 
+const getAltThemeOptions = (primaryColor: string): ThemeOptions => {
+  const themeOptions = getThemeOptions(primaryColor);
+  const altThemeOptions: ThemeOptions = {
+    typography: {
+      fontFamily: "Arial",
```

Comment:
in the meantime before this becomes a super polished theme, i like that it can be a prank alastair theme :smiling_imp:

Reply:
I was going to go Comic Sans but couldn't do it to myself 😂